internal/sqllite/query.go

```diff
@@ -0,0 +1,60 @@
+package sqllite
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+//go:generate mockgen -source=query.go -package=sqllite -destination=mock_query.go
+type Query interface {
+	GetOperatorVersions(bundleName string) ([]string, error)
+}
+
+type connection struct {
+	dbFile string
+}
+
+func NewQuery(dbFile string) Query {
+	return &connection{dbFile}
+}
+
+// bundleQuery run a command to get available versions in the operatorbundle table, where
+// the name starts by bundleName query
+func (q *connection) GetOperatorVersions(bundleName string) ([]string, error) {
```

Comment:
Who calls this function?

Reply:
As shown in description of the PR, this should be later called from the validation method, which needs to know the oeprator version and do appropriate validation based on it.